pegasus/sites.v3/code.org/public/about/careers/engineering.haml

```diff
@@ -0,0 +1,82 @@
+                        =hoc_s(:engineering_hero_desc)
+                    %a{href: "https://code.org/about/careers#roles ", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("accessibility_explore_title")}}
+                        =hoc_s(:call_to_action_open_positions)
+            .col-40.flex-container.justify-end
+                %img.hero-banner-basic__image{src: "/images/about/careers/engineering/hero-image.png", alt: ""}
+        .clear
+
+
+%section.engineering-team#accessibility-team
+    .wrapper
+        %h2=hoc_s(:engineering_team_title)
+        .action-block.action-block--two-col
+            %img{src: "/images/about/careers/engineering/careers_team.jpg", alt: hoc_s(:accessibility_explore_title), class: "col-50 action-block__image"}
+            .text-wrapper.col-50
+                %p.heading-sm=hoc_s(:engineering_team_card_text_1)
+                %p.heading-sm=hoc_s(:engineering_team_card_text_2)
+                %a{href: "https://code.org/about/careers#roles ", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("accessibility_explore_title")}}
+                    =hoc_s(:call_to_action_open_positions)
+        .clear
+
+= view :section_divider_line
+
+%section.our-team-text
+    .wrapper
+        %h2=hoc_s(:engineering_our_team_title)
+        %p=hoc_s(:engineering_our_team_desc)
+        .our-team-text__divider.our-team-text__divider--small
+        %div
+            %img.our-team-text__image.our-team-text__image--1{src: "/images/about/careers/engineering/our-team--1.jpg", alt: hoc_s(:engineering_our_team_sub_title_1)}
+            %h3.heading-md=hoc_s(:engineering_our_team_sub_title_1)
+            %p=hoc_s(:engineering_our_team_text_1, markdown: :inline, locals: {youtube_url: "https://www.youtube.com/watch?v=YMjRTjonlWg"})
+            .clear
+
+            .our-team-text__divider
+
+            %img.our-team-text__image.our-team-text__image--2{src: "/images/about/careers/engineering/our-team--2.jpg", alt: hoc_s(:engineering_our_team_sub_title_2)}
+            %h3.heading-md=hoc_s(:engineering_our_team_sub_title_2)
+            %p=hoc_s(:engineering_our_team_text_2)
+            .clear
+
+            .our-team-text__divider
+
+            %img.our-team-text__image.our-team-text__image--3{src: "/images/about/careers/engineering/our-team--3.jpg", alt: hoc_s(:engineering_our_team_sub_title_1)}
+            %h3.heading-md=hoc_s(:engineering_our_team_sub_title_3)
+            %p=hoc_s(:engineering_our_team_text_3)
+            .clear
+
+            .our-team-text__divider
+
+            %img.our-team-text__image.our-team-text__image--4{src: "/images/about/careers/engineering/our-team--4.jpg", alt: hoc_s(:engineering_our_team_sub_title_2)}
+            %h3.heading-md=hoc_s(:engineering_our_team_sub_title_4)
+            %p=hoc_s(:engineering_our_team_text_4)
+            .clear
+
+= view :section_divider_line
+
+%section.no-padding-bottom.faq
```

Comment:
The `.no-padding-bottom` class can be removed — it's ok to use the inherited spacing here and it's breaking on tablet:

<img width="777" alt="Screenshot 2023-08-14 at 10 01 01 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/0daa5213-ca67-4e9b-90d8-3324810f42e4">